htdocs/comm/action/class/api_agendaevents.class.php

```diff
@@ -171,6 +171,7 @@ public function index($sortfield = "t.id", $sortorder = 'ASC', $limit = 100, $pa
                 $obj = $db->fetch_object($result);
                 $actioncomm_static = new ActionComm($db);
                 if ($actioncomm_static->fetch($obj->rowid)) {
+                    $actioncomm_static->fetch_optionals();
```

Comment:
With code standardization, we started to include all the fetch_optionals inside the fetch().
So can you move it at end of the $actioncomm_static->fetch()  ?

Reply:
Hello.

I hope it'll be ok this time.